pkg/azurefile/azurefile_test.go

```diff
@@ -1632,3 +1632,97 @@ func TestGetFileShareClientForSub(t *testing.T) {
 		assert.Equal(t, tc.expectedError, err)
 	}
 }
+
+func TestGetNodeInfoFromLabels(t *testing.T) {
+	ctx := context.TODO()
+
+	// Test case where kubeClient is nil
```

Comment:
could you write unit test like this?

https://github.com/kubernetes-sigs/azurefile-csi-driver/blob/7756c1117eb5bf6eb2b79b3e17add568b796c9bc/pkg/azurefile/utils_test.go#L154-L198